bip-taro.mediawiki

```diff
@@ -47,7 +47,7 @@ This document is licensed under the 2-clause BSD license.
 Bitcoin, the first decentralized blockchain, realized early in its life
 several systems that attempted to represent arbitrary assets within the
 constraints of the Bitcoin system itself. Amongst the earliest of these systems
-was Counterparty (now known as OMNI), a meta-token protocol on top of Bitcoin
+was Mastercoin (now known as OMNI), a meta-token protocol on top of Bitcoin
```

Comment:
I'm still very confused on this tbh, I changed this after a former mastercoin/counterparty developer corrected my prior wording...

I'd still like to leave in the Counterparty reference as well, since I think that system is still in operation (?)

Reply:
Yes counterparty is still in operation, but they both used op_return method.
I guess the main differences are counterparty was more of a community, they did a proof of burn so no ICO no company behind it, all the original founders left early on and it was run by the community
Where as mastercoin/omni was more of a company
mastercoin seemed to only be used for tether where counterparty kind of started the NFT use case
i.e. rarepepes, collectibles, game tokens etc